sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/SyncBenchmark.java

```diff
@@ -114,6 +114,12 @@ public T apply(T o, Throwable throwable) {
         configuration = cfg;
         logger = LoggerFactory.getLogger(this.getClass());
 
+        if (configuration.isPartitionLevelCircuitBreakerEnabled()) {
+            System.setProperty("COSMOS.PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG", "{\"isPartitionLevelCircuitBreakerEnabled\": true, "
+                + "\"circuitBreakerType\": \"COUNT_BASED\","
```

Comment:
is this staled config?

Reply:
Thanks - good catch! Fixed in next iteration.